css/amp-story-player.css

```diff
@@ -0,0 +1,37 @@
+amp-story-player {
+  background: var(--story-player-poster, lightgrey);
+  width: 405px;
+  height: 720px;
+}
+
+amp-story-player::after {
+  content: " ";
+  position: absolute;
+  top: calc(50% - 38px);
+  left: calc(50% - 38px);
```

Comment:
32px?

Reply:
Can we change the box model to `border-box` to make this a bit easier?
cf https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#What_is_the_CSS_box_model